package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xwing-data2",
-  "version": "3.3.4",
+  "version": "3.5.0",
```

Comment:
should this also be incremented? I assumed, but not sure how if there's some auto generation or something I have to worry about because JS is the enemy.